client/components/product-expiration/README.md

```diff
@@ -25,6 +28,7 @@ Optional purchase date, used only when `isRefundable` provided.
 const expiryDateMoment = moment( product.expiry );
 <ProductExpiration
 	expiryDateMoment={ expiryDateMoment }
+	renewDateMoment={ renewDateMoment }
```

Comment:
(minor) You should probably add a line which sets `renewDateMoment` above (similar to the line that sets `expiryDateMoment`).

Reply:
Thanks. Implemented.